server.py

```diff
@@ -120,11 +222,11 @@ def community():
       # outputs
       'initial_supply': initial_supply,
       'results': [
-        'plot3-'+plot_name+'.png',
-        'plot4-'+plot_name+'.png',
-        'plot5-'+plot_name+'.png',
-        'plot6-'+plot_name+'.png',
-        'plot7-'+plot_name+'.png',
+        get_relative_session_dir() + '/' + 'plot3-'+plot_name+'.png',
```

Comment:
These tailing "/" are not needed since `get_relative_session_dir()` already include them.

Reply:
Good catch! Odd that it doesn't cause an error, I suppose it's still parsed.